### ISSUE TITLE
Sophos captive portal: breakage

### ISSUE THREAD

**Issue by garret1317** [uBlockOrigin/uAssets#19645]
### Prerequisites

- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I am running the latest version of uBlock Origin (uBO).

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

My college wifi's captive portal page shows no text, and the sign in button doesn't work. see the screenshots for a better explanation
The text and functionality is provided by some scripts which uBO blocks.
I believe the college uses sophos UTM, as the url is `utm.[redacted].ac.uk` (and the sophos branding)

here's some more info about it from the sophos docs:

`https://doc.sophos.com/nsg/sophos-firewall/19.5/help/en-us/webhelp/onlinehelp/AdministratorHelp/Authentication/WebAuthentication/index.html#multi-user-hosts`

<details><summary>troubleshooting info</summary>

```
uBlock Origin: 1.51.0
Firefox: 112
filterset (summary):
 network: 98796
 cosmetic: 39520
 scriptlet: 18346
 html: 1029
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 35395-89, 2m
  ublock-badware: 7956-0, 2m
  ublock-privacy: 577-0, 2m
  ublock-unbreak: 2177-0, 2m
  ublock-quick-fixes: 269-0, 2m
  easylist: 70282-706, 1m
  easyprivacy: 33211-718, 1m
  urlhaus-1: 5767-0, 1m
  plowe-0: 3725-2, 1m
filterset (user): [empty]
trustedset:
 added: [array of 1 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 1730 ms
 maxAssetCacheWait: 7 ms
```

</details>

### URL(s) where the issue occurs.

I'm afraid I can't really provide an URL, since it's only available on the college network.
best i can do without doxxing myself is `https://<IP address of Sophos Firewall>:8090/httpclient.html`
I think I know what needs to be unblocked though, fwiw

### Screenshot(s)

with ubo
https://github.com/uBlockOrigin/uAssets/assets/76261416/17178032-d659-4249-b6ce-d9968cce40eb

ubo off
https://github.com/uBlockOrigin/uAssets/assets/76261416/2c212127-ba81-4189-afaf-4fa69307f624

### uBO version

1.51.0

### Browser name and version

Firefox 112.0.2

### Settings

tested with new browser profile, so no changes

### Notes

I think it'll work if `/javascript/validation/httpclient.js` and `/javascript/cyberoamAjax.js` are unblocked. they don't appear to be doing anything nasty at a brief glance, just making the thing work

**Comment by mapx-** [comment]
- open the uBO's logger: https://github.com/gorhill/uBlock/wiki/The-logger
- repeat the steps to get the issue
- go back into the uBO's logger and then copy and paste here all blocked requests

**Comment by garret1317** [comment]
<details><summary>Logger output</summary>

```
+0
://utm.$from=~utm.akindi.com|~utm.arc.nasa.gov|~utm.edu|~utm.io|~utm.lol|~utm.md|~utm.openskies.sh|~utm.pansa.pl|~utm.pharos360.com|~utoronto.ca
--
utm.[redacted].ac.uk
1
get
font
https://utm.[redacted].ac.uk:8090/themes/SophosSans-Light.ttf
```
```
+0
://utm.$from=~utm.akindi.com|~utm.arc.nasa.gov|~utm.edu|~utm.io|~utm.lol|~utm.md|~utm.openskies.sh|~utm.pansa.pl|~utm.pharos360.com|~utoronto.ca
--
utm.[redacted].ac.uk
1
get
font
https://utm.[redacted].ac.uk:8090/themes/SophosSans-Regular.ttf
```
```
+0
://utm.$from=~utm.akindi.com|~utm.arc.nasa.gov|~utm.edu|~utm.io|~utm.lol|~utm.md|~utm.openskies.sh|~utm.pansa.pl|~utm.pharos360.com|~utoronto.ca
--
utm.[redacted].ac.uk
1
get
font
https://utm.[redacted].ac.uk:8090/themes/SophosSans-Light.woff
```
```
+0
://utm.$from=~utm.akindi.com|~utm.arc.nasa.gov|~utm.edu|~utm.io|~utm.lol|~utm.md|~utm.openskies.sh|~utm.pansa.pl|~utm.pharos360.com|~utoronto.ca
--
utm.[redacted].ac.uk
1
get
font
https://utm.[redacted].ac.uk:8090/themes/SophosSans-Regular.woff
```
```
+0
://utm.$from=~utm.akindi.com|~utm.arc.nasa.gov|~utm.edu|~utm.io|~utm.lol|~utm.md|~utm.openskies.sh|~utm.pansa.pl|~utm.pharos360.com|~utoronto.ca
--
utm.[redacted].ac.uk
1
get
font
https://utm.[redacted].ac.uk:8090/themes/SophosSans-Light.woff2
```
```
+0
://utm.$from=~utm.akindi.com|~utm.arc.nasa.gov|~utm.edu|~utm.io|~utm.lol|~utm.md|~utm.openskies.sh|~utm.pansa.pl|~utm.pharos360.com|~utoronto.ca
--
utm.[redacted].ac.uk
1
get
font
https://utm.[redacted].ac.uk:8090/themes/SophosSans-Regular.woff2
```
```
+0
://utm.$from=~utm.akindi.com|~utm.arc.nasa.gov|~utm.edu|~utm.io|~utm.lol|~utm.md|~utm.openskies.sh|~utm.pansa.pl|~utm.pharos360.com|~utoronto.ca
--
utm.[redacted].ac.uk
1
get
image
https://utm.[redacted].ac.uk:8090/images/logo-sophos.png
```
```
+0
://utm.$from=~utm.akindi.com|~utm.arc.nasa.gov|~utm.edu|~utm.io|~utm.lol|~utm.md|~utm.openskies.sh|~utm.pansa.pl|~utm.pharos360.com|~utoronto.ca
--
utm.[redacted].ac.uk
1
get
image
https://utm.[redacted].ac.uk:8090/favicon.ico
```
```
+0
://utm.$from=~utm.akindi.com|~utm.arc.nasa.gov|~utm.edu|~utm.io|~utm.lol|~utm.md|~utm.openskies.sh|~utm.pansa.pl|~utm.pharos360.com|~utoronto.ca
--
utm.[redacted].ac.uk
1
get
script
https://utm.[redacted].ac.uk:8090/javascript/validation/httpclient.js?ver=18609
```
```
+0
://utm.$from=~utm.akindi.com|~utm.arc.nasa.gov|~utm.edu|~utm.io|~utm.lol|~utm.md|~utm.openskies.sh|~utm.pansa.pl|~utm.pharos360.com|~utoronto.ca
--
utm.[redacted].ac.uk
1
get
script
https://utm.[redacted].ac.uk:8090/javascript/cyberoamAjax.js
```
```
+0
https://utm.[redacted].ac.uk:8090/httpclient.html
```
</details>
that's with the filter like this, hope thats alright

![image](https://github.com/uBlockOrigin/uAssets/assets/76261416/7e4ee703-ede8-47e9-a880-81f38e757d7e)

**Comment by mapx-** [comment]
you are saying 

```
https://utm.[redacted].ac.uk:8090/javascript/validation/httpclient.js
https://utm.[redacted].ac.uk:8090/javascript/cyberoamAjax.js
```

have to be whitelisted to fix the breakage

This easyprivacy filter is blocking the 2 scripts

`://utm.$domain=~utm.akindi.com|~utm.arc.nasa.gov|~utm.edu|~utm.io|~utm.lol|~utm.md|~utm.openskies.sh|~utm.pansa.pl|~utm.pharos360.com|~utoronto.ca`

to be fixed we have to know  `utm.[redacted].ac.uk`

**Comment by garret1317** [comment]
can I send it privately

**Comment by mapx-** [comment]
send it here: `axis.trivia@gmail.com`

**Comment by garret1317** [comment]
sent

**Comment by mapx-** [comment]
test the filters I sent you

**Comment by mapx-** [comment]
another test:

```
@@https://utm.*:8090/javascript/validation/httpclient.js
@@https://utm.*:8090/javascript/cyberoamAjax.js
```

**Comment by garret1317** [comment]
rather a moot point now, but yes the filters you sent me worked
thanks for the fix
